packages/patterns-react/src/patterns/LeadSpaceCentered/README.md

```diff
@@ -0,0 +1,75 @@
+# Lead Space Centered
+
+> The Lead Space Centered pattern is to be utilized within IBM.com.
+
+## Getting started
+
+Here's a quick example to get you started.
+
+```scss
+// yourapplication.scss
+@import '@carbon/type/scss/font-face/mono';
+@import '@carbon/type/scss/font-face/sans';
+@include carbon--font-face-mono();
+@include carbon--font-face-sans();
+```
+
+> 💡 Only import font's once per usage
+
+```javascript
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { LeadSpaceCentered } from '@carbon/ibmdotcom-patterns-react';
+import 'yourapplication.scss';
+import '@carbon/ibmdotcom-styles/scss/patterns/leadspace/index.scss';
```

Comment:
@kennylam @annawen1 I believe we're moving forward with having the style import in the `yourapplication.scss` file as the suggested approach, right?